### DISCUSSION TITLE
error: DBRowFetch????

### DISCUSSION THREAD

**Discussion by hoangvn79** [rkapl123/DBAddin#12] [General]
I use activeConnString as: Provider = Microsoft.Jet.OLEDB.4.0; Data Source = "H:\DBADDIN\pubs.MDB";
in cell K3: = DBRowFetch ("SELECT * FROM jobs"; activeConnString; K4: K11)
but no result, returned nothing
It should have a cell value of:        PMA42628M
                                                      Paolo
                                                      M
                                                      Accorti
                                                      13
                                                      35
                                                      0877
                                                        27-Aug-92
I don't know why???

**Comment by rkapl123** [comment]
Could you attach the file to this issue?

**Comment by hoangvn79** [comment]
I would like to send the attached file with the following link:
[https://drive.google.com/drive/folders/1huPV3-XlpCdraDJ9x7L4upyCl7XuhuON?usp=sharing](url)
file pubs.mdb  I put it at the address:  H:\DBADDIN\pubs.MDB
Can you check for errors?
at sheet name 0 cell K3 and K13
I applied yellow with the formula:
K3 = DBRowFetch ("SELECT * FROM employee"; activeConnString; K4: K11)
K13 = DBRowFetch ("SELECT * FROM employee where emp_id = 'A-C71970F'"; activeConnString; K16: K23)
The results all return nothing

**Comment by rkapl123** [comment]
I'm sorry, but I tested it locally and couldn't find a problem with that.
However, I've seen that date querying is a proble with access you need to state #dd/mm/yyyy# as a date literal, which is currenty not supported by the DBDate function. As a workaround you could use "where hire_date > #"&TEXT(B4;"dd/MM/yyyy")&"#" instead
My connection string is
Provider=Microsoft.ACE.OLEDB.12.0;Data Source="C:\Users\rolan\Downloads\pubs.MDB";Persist Security Info=False;
The result is:
![grafik](https://user-images.githubusercontent.com/7430763/59132621-fff5f980-8975-11e9-94e2-87c79f836696.png)

Are there any Problems with dblistfetch?

**Comment by rkapl123** [comment]
you could also try to right click and select "refresh data"

**Comment by hoangvn79** [comment]
![2019-06-08_05-14-03](https://user-images.githubusercontent.com/35492809/59136262-83324200-89ac-11e9-985d-c0a92f11fdbd.png)
when I right click and select "refresh data" is error!!!
result is no seem like you
Function dblistfetch is run well. 
Can you send me the DBaddin-AddIn.xll file you compiled for me to check?
I have checked file DBaddin-AddIn.xll I compiled  on other machines that have also resulted in nothing

**Comment by hoangvn79** [comment]
you can check  DBaddin-AddIn.xll  I compiled
https://drive.google.com/file/d/1XPtAKFVwFoGetjorGKrt5N2WmX-OibnD/view?usp=sharing

**Comment by rkapl123** [comment]
Dear hoangvn79!
I found some problems in the DBRowFetch mechanism, so you might give this one a try.
The feedback (especially in case of errors) needs still improving for this function, so stay tuned.
-regards,
Roland

**Comment by rkapl123** [comment]
I've also added the pubs.mdb and access to the testcases.
